packages/teams-js/src/public/meeting.ts

```diff
@@ -597,6 +620,39 @@ export namespace meeting {
     );
   }
 
+  /**
+   * @hidden
+   * Allows an app to get the verbose meeting details for the meeting
```

Comment:
Regarding the `@beta` tag, would the current `@internal` tag serve the same purpose? 

Or is `@beta` better, as it communicates that it's an API under development, but third parties can also try out the API

Reply:
They are related, but you want both.

`@internal` means "if you are calling this and you aren't a 1P app we make no guarantees it will work and in fact we almost guarantee that it won't"

`@beta` means "we reserve the right to make breaking changes to this function with no warning". Once your new function is checked in, working correctly, being used, and hasn't changed in a while you can come back in and remove the `@beta` tag, at which point you can't make breaking changes to it anymore until a new major version of the library is released.